proposals/0370-modifiers.rst

```diff
@@ -292,8 +394,12 @@ Unresolved Questions
   to consider this idea separate from the quite considerable complexity of `#242`_,
   and so I have made it a separate proposal.
 
+.. _`#242`: https://github.com/ghc-proposals/ghc-proposals/pull/242
+
 * This proposal floats the idea of ``%oneShot`` and ``%inline``, but these
   might fit better as pragmas than modifiers. In any case, they are not
   proposed concretely here and would be subject to a future proposal.
 
-.. _`#242`: https://github.com/ghc-proposals/ghc-proposals/pull/242
+* How does this interact with Template Haskell?
```

Comment:
I don't know TH very well. I think the options might be something like:

1. TH types don't change until a modifier that would need them to change is introduced (so not with this proposal). Inside a splice, I think that means

   * At most one modifier is allowed on an arrow
   * Modifiers elsewhere can be either ignored or can throw an error

2. TH types change to get a list of modifiers, any place where modifiers are allowed.